recipes/plugins_library/plugin_document.py

```diff
@@ -41,13 +41,12 @@ def on_submit(self, label: Dict, asset_id: str) -> None:
 
         n_issues = len(text_issues_array)
 
-        for i in range(n_issues):
-            self.kili.append_to_issues(
-                label_id=label["id"],
-                project_id=project_id,
-                text=text_issues_array[i],
-                object_mid=mid_issues_array[i],
-            )
+        self.kili.create_issues(
```

Comment:
You should add a condition on `n_issues`

Reply:
ok